### PR TITLE
Implement Vector::negative

### LINKED ISSUE #14: Add Vector::invert

### PR DESCRIPTION
**Pull request by jeremystucki**
Resolves #14

### CODE REVIEW

File: src/vector.rs [hunk 1]
```diff
@@ -155,6 +155,11 @@ impl Vector {
             y: rotated_y,
         }
     }
+
+    /// Negates the vector
```
Comment (janhohenheim) [inline src/vector.rs]: ", returning a vector with the same magnitude pointing in the opposite direction."

File: src/vector.rs [hunk 2]
```diff
@@ -713,4 +718,26 @@ mod tests {
         assert_nearly_eq!(vector.x, rotated_vector.x);
         assert_nearly_eq!(vector.y, rotated_vector.y);
     }
+
+    #[test]
+    fn negative_works_with_default_vector() {
```
Comment (janhohenheim) [inline src/vector.rs]: the correct terminology is [zero vector](http://mathworld.wolfram.com/ZeroVector.html)

File: src/vector.rs [hunk 3]
```diff
@@ -713,4 +718,26 @@ mod tests {
...
+        let vector = Vector::default();
+        let expected_vector = vector;
+
+        let inverted_vector = vector.inverted();
```
Comment (janhohenheim) [inline src/vector.rs]: shouldn't this be `.negative()`?

File: src/vector.rs [hunk 4]
```diff
@@ -713,4 +718,26 @@ mod tests {
...
+        assert_nearly_eq!(expected_vector.x, inverted_vector.x);
+        assert_nearly_eq!(expected_vector.y, inverted_vector.y);
+    }
+
+    #[test]
+    fn negative_works_with_5_and_negative_10() {
+        let vector = Vector { x: 5.0, y: -10.0 };
+        let expected_vector = Vector { x: -5.0, y: 10.0 };
+
+        let inverted_vector = vector.inverted();
```
Comment (janhohenheim) [inline src/vector.rs]: shouldn't this be `.negative()`?

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: src/vector.rs (hunk 3 above)
Comment (jeremystucki) [reply src/vector.rs]: c1144e5

File: src/vector.rs (hunk 4 above)
Comment (jeremystucki) [reply src/vector.rs]: c1144e5

File: src/vector.rs (hunk 2 above)
Comment (jeremystucki) [reply src/vector.rs]: c1144e5

File: src/vector.rs (hunk 1 above)
Comment (jeremystucki) [reply src/vector.rs]: ed7c0d2

**Review by janhohenheim — DISMISSED** [review]

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**